rest/data/patch.go

```diff
@@ -187,7 +187,7 @@ func AbortPatchesFromPullRequest(event *github.PullRequestEvent) error {
 		return err
 	}
 
-	if err = model.AbortPatchesWithGithubPatchData(*event.PullRequest.ClosedAt, true, "", owner, repo, *event.Number); err != nil {
+	if err = model.AbortPatchesWithGithubPatchData(*event.PullRequest.ClosedAt.GetTime(), true, "", owner, repo, *event.Number); err != nil {
```

Comment:
If we can update this to also not access pointers that would be cool but I see we already did that so alas

Reply:
draft